package.json

```diff
@@ -41,34 +41,36 @@
     "file-loader": "^1.1.5",
```

Comment:
Let's be careful not to make any version changes here unless they are necessary for your code to work.